python/samples/getting_started/00-getting-started.ipynb

```diff
@@ -20,6 +20,72 @@
     "%pip install semantic-kernel==1.1.0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Config\n",
+    "\n",
+    "Let's get started with the necessary configuration to run Semantic Kernel. For Notebooks, we require a `.env` file with the proper settings for the model you use. You may copy the `.env.example` file from this directory and place it either in this current directory, or in the main semantic-kernel/python directory.\n",
+    "\n",
+    "If interested, as you learn more about Semantic Kernel, there are a few other ways to make sure your secrets, keys, and settings are used:\n",
+    "\n",
+    "- Set the keys/secrets/endpoints as environment variables in your system.\n",
+    "- Manually configure the `api_key` or required parameters on either the `OpenAIChatCompletion` or `AzureChatCompletion` constructor with keyword arguments.\n",
+    "\n",
+    "For now we want to keep things as simple as possible. We will be more explicit and use the `dotenv` library to load the settings.\n",
```

Comment:
Remove this line. I think mentioning `dotenv` could add confusion without adding value.